### DISCUSSION TITLE
Need a translation string for "Default Shortcuts" window

### DISCUSSION THREAD

**Discussion by sh-cho** [rxhanson/Rectangle#589] [Ideas]
![image](https://user-images.githubusercontent.com/11611397/135875708-0a071816-6629-48f2-a430-4942dce1df0d.png)

I'm working on fixing some missing translation, but I can't find these strings on `Main.strings`.

https://github.com/rxhanson/Rectangle/blob/3cf4139d67c8e9bf109831602a1322931e137141/Rectangle/PrefsWindow/SettingsViewController.swift#L94

Well I find that here but don't know how to extract it cause I'm not familiar with swift...

**Comment by rxhanson** [comment]
Thanks! I just added this in as well - looks like I was previously being lazy.
